customer/src/main/java/com/yas/customer/controller/UserAddressController.java

```diff
@@ -34,4 +35,10 @@ public ResponseEntity deleteAddress(@PathVariable Long id) {
         userAddressService.deleteAddress(id);
         return ResponseEntity.ok().build();
     }
+
+    @PutMapping("/storefront/user-address/{id}")
+    public ResponseEntity chooseDefaultAddress(@PathVariable Long id) {
+        userAddressService.chooseDefaultAddress(id);
+        return ResponseEntity.ok().build();
```

Comment:
return type should be ResponseEntity<Void>

return statement should be return ResponseEntity.noContent().build();